pkg/detectors/detectors.go

```diff
@@ -57,6 +57,10 @@ type Result struct {
 	Redacted       string
 	ExtraData      map[string]string
 	StructuredData *detectorspb.StructuredData
+
+	// This field should only be populated if the verification process itself failed in a way that provides no
+	// information about the verification status of the candidate secret, such as if the verification request timed out.
+	VerificationErrorMessage string
```

Comment:
What do you think about using an `error` here? It makes slightly more sense from a types perspective in my opinion, and a `string` can be retrieved from it with the `Error()` method.

Reply:
seems prudent